schemas/IDLs/org.gel.models.metrics.avro/1.0.1/ReportedVsGenetic.avdl

```diff
@@ -0,0 +1,153 @@
+@namespace("org.gel.models.report.avro")
```

Comment:
this should probably be `@namespace("org.gel.models.metrics.avro")` -- what do you think @priesgo @Antonior26  ?

Reply:
indeed

Reply:
Done